docs/src/main/paradox/akka-management.md

```diff
@@ -40,7 +40,7 @@ actually want to use (and load) in their project.
 
 @@dependency[sbt,Gradle,Maven] {
   group="com.lightbend.akka.management"
-  artifact="akka-management"
+  artifact="akka-management_$scala.binaryVersion$"
```

Comment:
Though a bit weird, I though this just works hm... perhaps it only does in Akka, I will have a look later on why that is.